addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/SubResourceIntegrityAttributeScanRule.java

```diff
@@ -57,6 +58,10 @@ public class SubResourceIntegrityAttributeScanRule extends PluginPassiveScanner
     private static final Logger LOGGER =
             LogManager.getLogger(SubResourceIntegrityAttributeScanRule.class);
 
+    // https://html.spec.whatwg.org/multipage/semantics.html#attr-link-integrity
+    private static final List<String> LINK_REL_KEYWORDS_FOR_SRI =
```

Comment:
Consider using a `Set<String>` since we're only checking membership, not iterating.